lang/java/reef-applications/reef-vortex/README.md

```diff
@@ -0,0 +1,17 @@
+Vortex
+========================
+Vortex is a runtime for low-priority jobs to effectively cope 
```

Comment:
This description is an important application of Vortex, but it does not define what Vortex is. Can you add descriptions about tasklet abstraction, queue, and the thread API and then talk about low-priority and high-priority jobs briefly? We don't need to talk much about the latter since we're in the process of exploring.